modules/tests/README.md

```diff
@@ -5,9 +5,11 @@
 This directory contains the model-based tests for the IBC modules. They are "model-based" because they are generated from a `TLA+` model of the IBC modules (see [IBC.tla](support/model_based/IBC.tla)).
 
 To instantiate the model, we define in [IBC.cfg](support/model_based/IBC.cfg) the following model constants:
-    - `ChainIds = {"chain-A", "chain-B"}`, indicating that two chains, `chain-A` and `chain-B`, will be created
-    - `MaxClientsPerChain = 1`, indicating that at most 1 client per chain will be created
-    - `MaxClientHeight = 2`, indicating that clients will reach at most height 2
+
+- `ChainIds = {"chain-A", "chain-B"}`, indicating that two chains, `chain-A` and `chain-B`, will be created
```

Comment:
The change in 5d6bf08 was reverted because the model doesn't know about revisions. Also, when generating the actual `ChainId` on the Rust side, we would get `"chain-A-0-0"`:
https://github.com/informalsystems/ibc-rs/blob/3e513dbfd96fca6de0b2507bab5baff936b9864d/modules/tests/executor/mod.rs#L96-L98